conseil-api/src/main/scala/tech/cryptonomic/conseil/api/config/ConseilAppConfig.scala

```diff
@@ -0,0 +1,181 @@
+package tech.cryptonomic.conseil.api.config
+
+import com.github.ghik.silencer.silent
+import com.typesafe.config.{ConfigObject, ConfigValue}
+import com.typesafe.scalalogging.LazyLogging
+import pureconfig.error.{ConfigReaderFailures, ExceptionThrown, FailureReason}
+import pureconfig.generic.ProductHint
+import pureconfig.generic.auto._
+import pureconfig.{ConfigReader, _}
+import scopt.OptionParser
+import tech.cryptonomic.conseil.api.security.Security
+import tech.cryptonomic.conseil.common.config.Platforms._
+import tech.cryptonomic.conseil.api.security.Security._
```

Comment:
Can I ask you to kindly put imports sorted to a minimum, to make it easier to figure out?
Just putting similar things together, no need for anything too strict.
Thanks

Reply:
Sure, I guess `scalafmt` does not optimize them, cause I just did it and still issue is there. I did it manually here, and will try to find the places where it's unoptimized or find out more automatic approach.

Reply:
Ok, yeah, scalafmt doesn't help, and I'm sorry to say that I generally clean up things manually...
Let's say that it might help the reader, to easily find imports, but it's not fundamental in any way.

Reply:
Yea, I'm mostly doing it too, but there are couple of places where imports are marked as 'unused' but they are actually used, so IntelliJ goes crazy, removes them and I'm back with the weird compilation issues. I'm marking these imports as used, just for me, but clearly forgot to do it in this class. Will pay attention in the future - this one is done.